packages/layout/layout.jade

```diff
@@ -0,0 +1,7 @@
+head
+  title Mantle
+
+template(name='layout')
+  +header
+  +Template.dynamic template=main data=data
```

Comment:
I'm not sure what to do about the data argument here. I included it because I think it could be a useful way of getting URL parameters to templates. flow-router recommends not passing any data from the router to templates, and calling FlowRouter.getParam instead, but then the templates have a dependency on the router and they need to be different depending on whether they're called from the router or from another template. I agree about having templates manage their own data, but I want to be able to pass in just the URL parameters. Maybe I should at least rename the variable used in the routes to something other than 'data'. I'm also not sure if using this data argument will have any reactivity implications when the app gets more complex.